### PR TITLE
Bugfix: Create app now uses proper dimensions on create

### LINKED ISSUE #380: Zones are resized to the size of a cube when spawned
I noticed that the zone starts off larger than me and flashes into the smaller size before happening, perhaps this was not intended functionality by High Fidelity?

### PR DESCRIPTION
**Pull request by kasenvr**
This problem was introduced with this commit https://github.com/kasenvr/project-athena/commit/2fc0f01928bf94bf1bd3dfafbd7b1b84c4d68ae9#diff-48b43e48f4b869756959f7bd81c1887a 

Closes #380

### CODE REVIEW

File: scripts/system/create/edit.js [hunk 1]
```diff
@@ -613,10 +613,18 @@ var toolBar = (function () {
...
                 isLoadedCheckCount++;
                 if (isLoadedCheckCount === MAX_LOADED_CHECKS || Entities.isLoaded(entityID)) {
-                    var naturalDimensions = Entities.getEntityProperties(entityID, "naturalDimensions").naturalDimensions
+                    var dimensionsToUse;
+                    if (POST_ADJUST_ENTITY_TYPES.indexOf(properties.type) !== -1) {
+                        // If it is a "Model", use natural dimensions...
+                        dimensionsToUse = Entities.getEntityProperties(entityID, "naturalDimensions").naturalDimensions;
+                    } else {
+                        // If it is not a "Model", use local dimensions...
+                        dimensionsToUse = Entities.getEntityProperties(entityID, "localDimensions").localDimensions;
```
Comment (HifiExperiments) [inline scripts/system/create/edit.js:622]: Do we even need this for non-model entities?  Can we just make it so this loaded check stuff is only called for models?

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR406/ProjectAthena-Alpha-PR406-20261f7_3c21a4d3.dmg

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR406/ProjectAthena-Alpha-PR406-d0497b9_778276bb.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR406/ProjectAthena-Alpha-PR406-d0497b9.tar_778276bb.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR406/ProjectAthena-Alpha-PR406-d0497b9.tar_778276bb.Z

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR406/ProjectAthena-Alpha-PR406-d0497b9_778276bb.exe

**Comment by ctrlaltdavid** [comment]
The problem commit was merged into HiFi master in 3b9a8b4, "dev-2718/models-brought-in-at-incorrect-dimensions".

**Comment by AleziaKurdis** [comment]
Entities initial size is back to normal with this PR.
Tested: Zone, Shape, Image, Text, Web, Material, Particle, Light, Modelé

**Comment by SilverfishVR** [comment]
tested _ProjectAthena-Alpha-PR406-d0497b9_778276bb.exe_
yep, looks good to me too 👍

**Pull request merged**